app/controllers/admin/calendars_controller.rb

```diff
@@ -53,23 +55,27 @@ def destroy
       authorize @calendar
       @calendar.destroy
       respond_to do |format|
-        format.html { redirect_to admin_calendars_url, notice: 'Calendar was successfully destroyed.' }
+        format.html do
+          flash[:success] = 'Calendar was successfully destroyed.' 
```

Comment:
change this to deleted like the others and otherwise looking great!

Reply:
we should probably also note if events on the calendar get deleted too - i don't remember what happens currently

Reply:
Is this handled by the Events controller?